### PR TITLE
Pick up 1.90

### PR DESCRIPTION
**Pull request by Mendor**
Life critical fix because of https://github.com/kohsuke/github-api/issues/387

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
   </licenses>
 
   <artifactId>github-api</artifactId>
-  <version>1.90-SNAPSHOT</version>
+  <version>1.91-SNAPSHOT</version>
```
Comment (KostyaSha) [inline pom.xml]: don't change it

Comment (Mendor) [reply pom.xml]: Thanks, fixed.

**Comment by OleksandrBerezianskyi** [comment]
@Mendor @KostyaSha any chance to get this built? There are no meaningful errors in the build log https://ci.jenkins.io/blue/rest/organizations/jenkins/pipelines/Plugins/pipelines/github-api-plugin/branches/PR-16/runs/1/nodes/7/log/?start=0 , it seems that the job was just terminated.

**Comment by nwillems** [comment]
I've made this https://github.com/Mendor/github-api-plugin/pull/1 <- Hope it helps.

**Comment by SqREL** [comment]
Hi @Mendor could you please merge this, since all checks have passed

**Comment by SqREL** [comment]
@KostyaSha could you please review and merge this PR since all checks have passed

**Comment by bjoernhaeuser** [comment]
I am using this, but still getting exception when using the github commit status setter. Though currently still trying to understand if this is actually the problem or not.

**Comment by mamantoha** [comment]
@bjoernhaeuser this is actually the problem, yes.

**Comment by egormasalitin** [comment]
@KostyaSha Any chance to merge it today?

**Comment by jglick** [comment]
Picks up https://github.com/kohsuke/github-api/pull/388 IIUC.

**Pull request merged**